src/services/side-panels/ui/icon-wrapper/icon-wrapper.component.tsx

```diff
@@ -0,0 +1,31 @@
+import { h, Component, ComponentChild, RefObject, cloneElement, VNode } from 'preact';
+
+type ToggleState = {
+  on: boolean;
+};
+
+type ToggleProps = {
+  ref: RefObject<IconWrapper>;
```

Comment:
where you use this ref?

Reply:
[here](https://github.com/kaltura/playkit-js-ui-managers/blob/603a48493fed36f913dcce50eb6126dd7581ad0c/src/services/side-panels/ui/icon-wrapper/icon-wrapper.component.tsx#L16)

Reply:
[here](https://github.com/kaltura/playkit-js-ui-managers/blob/603a48493fed36f913dcce50eb6126dd7581ad0c/src/services/side-panels/ui/icon-wrapper/icon-wrapper.component.tsx#L16)